<?xml version="1.0"?>
<launch>
  <arg name="paused" default="false"/>
  <arg name="use_sim_time" default="true"/>
  <arg name="gui" default="true"/><arg name="headless" default="false"/>
  <arg name="debug" default="false"/>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find multi_robot)/worlds/flood_circle.world"/>
    <arg name="paused" value="$(arg paused)"/>
    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
    <arg name="gui" value="$(arg gui)"/>
    <arg name="headless" value="$(arg headless)"/>
    <arg name="debug" value="$(arg debug)"/>
  </include>
  <group ns="drone14">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
  <arg name="name" default="drone14"/>
  <arg name="tf_prefix" value="drone14"/>
  <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_downward_cam_red.gazebo.xacro"/>
  <arg name="x" default="-14.0"/>
  <arg name="y" default="14.0"/>
  <arg name="z" default="0.2"/>
  </include>
  </group>
</launch>